### PR TITLE
Add images directory link to readme.

### PR DESCRIPTION
**Pull request by juanfra**
**Description**

Adding a link to the public Google Drive folder where the theme images live, as discussed in https://github.com/WordPress/twentytwentyfive/issues/135
https://tinyurl.com/tt5-images

**Screenshots**

N/A

**Testing Instructions**

- Check the readme.
- Click on https://tinyurl.com/tt5-images
- Confirm the images are there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,8 @@ These functions enhance security and support localization efforts in WordPress b
 
 - **Patterns with images**
 
+You can find the images used in the theme designs, in the following link: https://tinyurl.com/tt5-images
```
Comment (jasmussen) [inline README.md]: Suggestion:

`You can find the images used in the theme designs, [in this folder](https://tinyurl.com/tt5-images).`

**Review by jasmussen — APPROVED** [review]
Small suggestion for making it a nicer link, otherwise, good to go!

**Comment by huzaifaalmesbah** [comment]
Thank you for the excellent work @juanfra ! Everything looks good to me, and this has been incredibly helpful.

**Comment by kafleg** [comment]
Where I can find the license and source link of those images? Once the images are bundled inside the theme, we need to mention the license of all the images used in the readme file.

**Comment by jasmussen** [comment]
@kafleg 

> Where I can find the license and source link of those images? Once the images are bundled inside the theme, we need to mention the license of all the images used in the readme file.

For now, [here in Figma](https://www.figma.com/design/dzGCSntVch4EQdVERTqyVK/Twenty-Twenty-Five?node-id=77-8118&t=0tJE2VXhthA9HmZK-1). However those attributions are being moved to the README, [discussion here](https://github.com/WordPress/twentytwentyfive/issues/135#issuecomment-2314760179). @juanfra correct me if I'm wrong.

**Comment by juanfra** [comment]
Yes, the image links live in Figma.

@kafleg Almost all images come from rawpixel. The filename includes the ID, and the way to get the license is by following the link rawpixel.com/image/$id. That way, when a contributor is working on a pattern where an image is used, they need to:

- Get the image from the Google Drive folder -> https://tinyurl.com/tt5-images
- Look for the id in the filename.
- Visit `rawpixel.com/image/$id`, replacing `$id` with the value you get from the filename.
- Get the license.
- Put it in the `readme.txt` file.

**Comment by kafleg** [comment]
> Yes, the image links live in Figma.
> 
> @kafleg Almost all images come from rawpixel. The filename includes the ID, and the way to get the license is by following the link rawpixel.com/image/$id. That way, when a contributor is working on a pattern where an image is used, they need to:
> 
> * Get the image from the Google Drive folder -> https://tinyurl.com/tt5-images
> * Look for the id in the filename.
> * Visit `rawpixel.com/image/$id`, replacing `$id` with the value you get from the filename.
> * Get the license.
> * Put it in the `readme.txt` file.

I found images of teams are from Flickr. I added license of those images on  #126 Please check it.

**Review by jasmussen — APPROVED** [review]
Fine to ship for now. Eventually we'll want the actual full URLs in the README, not instructions. But this is a good step forward, thank you!

I've still prodded people to double check the Flickr images, I still assume they are fine to use, but I'll report back when I hear news.

**Comment by juanfra** [comment]
Thanks, Joen. Yes, I would have included them, but there are more images on the Figma page than we use for the theme. That means we would have had to map which ones we included or not, and it'd have been really time-consuming.

These instructions will help contributors add the license on-demand for now.

Thanks for the reviews ✨

**Pull request merged**